### DISCUSSION TITLE
Charges only capture parts of the charge

### DISCUSSION THREAD

**Discussion by amueller-tech** [teslamate-org/teslamate#3121] [General]
Hi,

I have an issue capturing a full charge in the charge statistics with the latest teslamate version.
While the charge shows up in the overview, the statistics only show parts of it.

- Charge started at 6pm at 13% SOC

- Perfectly shows in the Overview section under charging details, also the states and the charge level progression is correct:
<img width="592" alt="image" src="https://user-images.githubusercontent.com/57542733/218466598-4fb07f12-b00a-46de-9c94-917cd435d2c8.png">

- Under "charges" the charge started at 2am at 40%, skipping the first half of the charge
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/57542733/218467027-4a381c15-b864-4b5b-88f8-8c465241f354.png">

**Comment by amueller-tech** [comment]
might be related to this one: https://github.com/adriankumpf/teslamate/discussions/2485,
but my charging details are actually correct, only the stats are incomplete

**Comment by pacioc193** [comment]
i've same issue

Charges data
![image](https://user-images.githubusercontent.com/3874577/229218742-7ebd4186-6bd1-40c0-9492-5b34552dc24a.png)

Charging status
![image](https://user-images.githubusercontent.com/3874577/229218853-ab72ebb3-9fa3-48d0-93dc-a211cf620d56.png)
